Packs/Base/Scripts/GetMLModelEvaluation/GetMLModelEvaluation.py

```diff
@@ -207,7 +221,8 @@ def find_threshold(y_true_str, y_pred_str, customer_target_precision, target_rec
     entry = output_report(np.array(y_true), y_true_per_class, np.array(y_pred), y_pred_per_class, unified_threshold,
                           customer_target_precision, unified_threshold_precision, detailed_output)
     per_class_entry = calculate_per_class_report_entry(class_to_arrs, labels, y_pred_per_class, y_true_per_class)
-    return [entry, per_class_entry]
+    res = merge_entries(entry, per_class_entry)
```

Comment:
did you check it manually? is it looks ok?

Reply:
yes, seems like a one entry

Reply:
![image](https://user-images.githubusercontent.com/57318201/98518608-44c34580-2278-11eb-9bc7-879bd9c9c3bb.png)